test/doc/bessel_h.cpp

```diff
@@ -0,0 +1,24 @@
+#include <kyosu/kyosu.hpp>
+#include <eve/wide.hpp>
+#include <iostream>
+
+int main()
+{
+  std::cout.precision(16);
+  using w_t = eve::wide<double, eve::fixed<2>>;
+  auto z = kyosu::complex(w_t(20.0, 1.5), w_t(0.0, 1.5));
+  auto v = -(2+1/3.0);
+  int nb = int(eve::abs(v)+1);
+  std::cout << "z                    " << z << std::endl;
+  std::vector<decltype(z)> hs(nb);
```

Comment:
you now the type of z, so use it.

Reply:
no, I have not computed the type of z. I can do it but it's more verbose
I do :  auto z = kyosu::complex(w_t(20.0, 1.5), w_t(0.0, 1.5));